README.md

```diff
@@ -2,46 +2,58 @@
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
-# Azure Terraform Module Generator
-`generator-az-terra-module` creates base template to start a new terraform module.
+## Azure Terraform Module Generator
 
-![Terraform module template walkthrough](/doc/scaffolding.640x434.gif)
+`generator-az-terratest-module` creates base template to start a new terraform module along with terratest for automatically testing the module.
```

Comment:
>`generator-az-terratest-module` [](start = 0, length = 31)

Don't change the name of generator:
1. Terratest is just some kind of testing suite/framework, no need to make the generator such special.
2. Change name may invalidate legacy users and impact SEO.

Reply:
Yes as I have mentioned in the pull request, it is another package I have published in my account. Of course I will use the old name later.